pkg/internal/cidr/cidr_test.go

```diff
@@ -34,20 +34,20 @@ func TestSplitIntoSubnetsIPv4(t *testing.T) {
 		{
 			// https://aws.amazon.com/about-aws/whats-new/2018/10/amazon-eks-now-supports-additional-vpc-cidr-blocks/
 			name:        "default secondary cidr block configuration with primary cidr",
-			cidrblock:   "100.64.0.0/16",
+			cidrblock:   "100.64.0.0/10",
```

Comment:
Neat. Nice find.... Technically, those values could be anything. But it's nice to reflect the documentation.